src/OpenTelemetry/Metrics/MeterProviderBuilderExtensions.cs

```diff
@@ -37,6 +37,54 @@ public static MeterProviderBuilder SetObservationPeriod(this MeterProviderBuilde
             return null;
         }
 
+        /// <summary>
+        /// Sets export period.
+        /// </summary>
+        /// <param name="meterProviderBuilder"><see cref="MeterProviderBuilder"/>.</param>
+        /// <param name="periodMilliseconds">Perion in milliseconds.</param>
+        /// <returns><see cref="MeterProvider"/>.</returns>
+        public static MeterProviderBuilder SetExportPeriod(this MeterProviderBuilder meterProviderBuilder, int periodMilliseconds)
+        {
+            if (meterProviderBuilder is MeterProviderBuilderSdk meterProviderBuilderSdk)
+            {
+                return meterProviderBuilderSdk.SetExportPeriod(periodMilliseconds);
+            }
+
+            return null;
```

Comment:
How to handle the error in caller if this method returns `null`?

Reply:
return the provided meterProviderBuilder. I made copy-paste errors above ;)